packages/desktop-client/src/components/Titlebar.tsx

```diff
@@ -114,6 +114,7 @@ function PrivacyButton({ style }) {
   return (
     <Button
       type="bare"
+      aria-label="Toggle Privacy"
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏would be better to be explicit here for visually impaired folks.

```suggestion
      aria-label={`${isPrivacyEnabled ? 'Disable' : 'Enable'} privacy mode`}
```